Add README (#2)
populationgenomics/ancestry: README.md

# ancestry

Prioritising and assigning ancestry to populations.

## Running hail

> To connect  to hail with the required gnomAD dependencies, run the following command:

Reviewer: ```suggestion
To connect to hail with the required gnomAD dependencies, run the following command:
```

> ```hailctl dataproc start --max-age 2h --region australia-southeast1 my-cluster --packages gnomad```

Reviewer: If you want to split the command up a bit for more clarity, you can use `\` at the end of each line - bash will treat the next line as a continuation of the command:

```
hailctl dataproc start \
  --max-age 2h \
  --region australia-southeast1 \
  --packages gnomad \
  my-cluster 
```

> Some commands are computationally expensive and require more workers. For these jobs, you can modify the cluster. I specify 100 preemptible workers with the following command:

Reviewer: ```suggestion
Some commands launch jobs that are computationally expensive and require more workers. For these jobs, you can modify the cluster. I specify 100 preemptible workers with the following command:
```

> ```hailctl dataproc modify my-cluster --num-preemptible-workers 100 --num-workers 2```

Reviewer: Fyi, you can get away with just using a single backtick when wanting in-line code in markdown:

```suggestion
`hailctl dataproc modify my-cluster --num-preemptible-workers 100 --num-workers 2`
```

To stop the cluster, run the following:

```hailctl dataproc stop```

## Using and viewing Jupyter notebooks

To launch a Jupyter notebook, run the following command:

```hailctl dataproc connect my-cluster nb```

> Not all images in Jupyter notebooks are able to be viewed on Github. In order to view all of the Jupyter notebook images, use nbviewer instead of clicking on the Jupyter notebook file within this Github repository.

Reviewer: ```suggestion
GitHub renders Jupyter notebooks as static HTML files, so _interactive_ plots [cannot](https://docs.github.com/en/github/managing-files-in-a-repository/working-with-jupyter-notebook-files-on-github) be viewed directly through the repository. In order to view all the interactive features of the notebook, you can enter the location of the `.ipynb` file in the [nbviewer](https://nbviewer.jupyter.org/).
```

> To view the script HGDP_1KG_HailExploration.ipynb, (click here.)[https://nbviewer.jupyter.org/github/populationgenomics/ancestry/blob/main/scripts/HGDP_1KG_HailExploration.ipynb]

Reviewer: Square brackets first, parentheses second (I used to make this mistake all the time) ;-)

```suggestion
To view the HGDP_1KG_HailExploration.ipynb Jupyter notebook, [click here.](https://nbviewer.jupyter.org/github/populationgenomics/ancestry/blob/main/scripts/HGDP_1KG_HailExploration.ipynb)
```
